hr_holidays_accrual/views/hr_leave_allocation.xml

```diff
@@ -0,0 +1,129 @@
+                <attribute name="attrs">
+                    {
+                        'invisible': [('accrual', '=', True)]
+                    }
+                </attribute>
+            </xpath>
+            <xpath expr="//label[@for='number_of_days']/following-sibling::*[1]" position="attributes">
+                <attribute name="attrs">
+                    {
+                        'invisible': [('accrual', '=', True)]
+                    }
+                </attribute>
+            </xpath>
+            <xpath expr="//label[@for='date_to']" position="replace" priority="20"/>
+            <xpath expr="//field[@name='date_to']/../.." position="replace" priority="20"/>
+            <xpath expr="//field[@name='accrual']/../..">
+                <group attrs="{'invisible': [('accrual', '=', False)]}">
+                    <field name="date_from"/>
+                    <field name="date_to"/>
+
+                    <label for="limit_accrued_days"/>
+                    <div class="o_row">
+                        <field name="limit_accrued_days"/>
+                        <field name="max_accrued_days" attrs="{'invisible': [('limit_accrued_days', '=', False)]}"/>
+                    </div>
+
+                    <label for="limit_carryover_days"/>
+                    <div class="o_row">
+                        <field name="limit_carryover_days"/>
+                        <field name="max_carryover_days" attrs="{'invisible': [('limit_carryover_days', '=', False)]}"/>
+                    </div>
+
+                    <label for="limit_accumulated_days"/>
+                    <div class="o_row">
+                        <field name="limit_accumulated_days"/>
+                        <field name="max_accumulated_days" attrs="{'invisible': [('limit_accumulated_days', '=', False)]}"/>
+                    </div>
+
+                    <field name="accrual_method"/>
+
+                    <label for="number_per_interval"/>
+                    <div class="o_row">
+                        <field name="number_per_interval" class="ml8"/>
+                        <field name="unit_per_interval"/>
+                    </div>
+
+                    <label for="interval_number"/>
+                    <div class="o_row">
+                        <field name="interval_number" class="ml8"/>
+                        <field name="interval_unit"/>
+                    </div>
+                </group>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="hr_leave_allocation_view_form" model="ir.ui.view">
+        <field name="name">hr.leave.allocation.view.form.inherit.accrual</field>
+        <field name="model">hr.leave.allocation</field>
+        <field name="inherit_id" ref="hr_holidays.hr_leave_allocation_view_form"/>
+        <field name="arch" type="xml">
+            <header position="inside">
+                <button
+                    name="%(action_hr_leave_allocation_as_of_date)d"
```

Comment:
module_name.action_id (Not sure about this one)

Reply:
while within same module, it's ok to refer like that